lib/Controller/PageController.php

```diff
@@ -91,6 +92,12 @@ public function index(): TemplateResponse {
 		Util::addStyle(Application::APP_ID, 'icons');
 
 		$response = new TemplateResponse(Application::APP_ID, 'main');
+
+		$policy = new ContentSecurityPolicy();
+		$policy->addAllowedWorkerSrcDomain("'self'");
+		$policy->addAllowedScriptDomain('blob:');
```

Comment:
Safe enough @rullzer ?

Reply:
how is the script all of a sudden a blob?

Reply:
I followed this, but I may be totally wrong https://medium.com/@krishnachirumamilla/content-security-policy-worker-src-cd06ecfa2fe8

Reply:
@rullzer I updated it to be "self"

Reply:
ah ok I know why it is confusing me. Normally we only run it with nonce.
Self is a lot better yeah.
And we can always thighen it later on.